aspnet/mvc/views/overview.rst

```diff
@@ -0,0 +1,79 @@
+Views Overview
+==============
+
+By `Steve Smith`_
+
+ASP.NET MVC Core controllers can return formatted results using *views*.
+
+.. contents:: Sections
+  :local:
+  :depth: 1
+
+What are Views?
+---------------
+
+In the Model-View-Controller (MVC) pattern, the *view* encapsulates the presentation details of the user's interaction with the app. Views are templates that include HTML with embedded code that generates content to send to the client. The code uses :doc:`razor syntax <razor>`, which have deep understanding of HTML that allows it to interact with the HTML in the document with minimal code or ceremony.
+
+ASP.NET Core MVC views are *.cshtml* files stored by default in a *Views* folder within the application. Typically, each controller will have its own folder, in which are views for each of the controller actions that returns a view. 
+
+.. image:: overview/_static/views_solution_explorer.png
+
+In addition to action-specific views, :doc:`partial views <partial>`, :doc:`layouts, and other special view files <layout>` can be used to help reduce repetition and allow for reuse within the app's views.
+
+Creating a View
+^^^^^^^^^^^^^^^
+
+To create a view, first navigate to the */Views* folder of your app. If the view will be used by multiple actions, you may want to create it in the *Shared* folder. Otherwise, create or open a folder for the controller that houses the action that will return the view. Name the file the same as the action, and add a *.cshtml* file extension. For example, to create a view for the *List* action on the *Products* controller, you would create the *List.cshtml* file in the */Views/Products* folder.
```

Comment:
Views that are specific to a controller are created in the *Views/<Controller>* folder. Views that are shared among controllers are ...

No need to tell them to navigate.

The root name of Razor view files are generally have the same name as the action method with a .cshtml extension.